eng/Publishing.props

```diff
@@ -6,5 +6,12 @@
     <!-- Set PlatformName to TargetArchitecture to create unique build manifest files. -->
     <PlatformName Condition="'$(TargetArchitecture)' != ''">$(TargetArchitecture)</PlatformName>
   </PropertyGroup>
+  
+  <!-- Update all Artifacts with Kind=Package to have additional metadata item Category="ToolingPackage".
+        This will mean that in VMR builds, deployment tool packages can get published to a different feed depending on
+        the channel configuration. -->
+   <ItemGroup>
+     <Artifact Update="@(Artifact->WithMetadataValue('Kind', 'Package'))" Category="ToolingPackage" />
```

Comment:
Is `ToolingPackage` an arbitrary name to match packages from `deployment-tools` repo? I presume that it is not related to a tool package that we build in repo, but not in VMR - dotnet-mage.

Reply:
It's a publishing category: https://github.com/dotnet/arcade/blob/80d7b4ee29c31c0886b139ad02d85da48b543cd6/src/Microsoft.DotNet.Build.Tasks.Feed/src/model/TargetFeedConfig.cs#L169